modules/admin/con-rbac-rest-api.adoc

```diff
@@ -74,6 +91,28 @@ The RBAC REST API supports the following HTTP status codes to return as response
 * `403` Forbidden: refusal to authorize request
 * `404` Not Found: could not find requested resource
 * `409` Conflict: request conflict with the current state and the target resource
+--
 
+Source::
++
+--
+Each permission policy and role created using the RBAC plugin is associated with a source to maintain data consistency within the plugin. You can manipulate permission policies and roles based on the following designated source information:
 
+* CSV file
+* Configuration file
+* REST API
+* Legacy
 
+Managing roles and permission policies originating from CSV files and REST API involves straightforward modification based on their initial source information.
+
+The Configuration file pertains to the default `role:default/rbac_admin` role provided by the RBAC plugin. The default role has limited permissions to create, read, update, and delete permission policies or roles, and to read catalog entities. 
+
+[NOTE]
+====
+In case the default permissions are insufficient for your administrative requirements, you can create a custom admin role with required permission policies.
+====
+
+The legacy source applies to policies and roles defined before RBAC backend plugin version `2.1.3`, and is the least restrictive among the source location options. You must update the permissions and roles in legacy source to use either REST API or the CSV file sources. 
```

Comment:
We capitalize RBAC backend plugin inconsistently throughout the docs. If the official name of the plugin is RBAC Backend plugin, then capitalize. If not, let's create a Jira ticket to make this capitalization consistent.